app/middleware/postcodeLookup.js

```diff
@@ -1,26 +1,35 @@
 const postcodeValidator = require('../lib/postcodeValidator');
+const renderer = require('./renderer');
 const log = require('../lib/logger');
 const PostcodesIOClient = require('postcodesio-client');
 
 const PostcodesIO = new PostcodesIOClient();
 
+function outsideEngland(outcodeDetails) {
+  return !outcodeDetails.country.some(c => c === 'England');
+}
+
 function lookupPostcode(req, res, next) {
   const postcode = res.locals.postcode;
 
   if (postcodeValidator.isOutcode(postcode)) {
     PostcodesIO.outcode(postcode, (err, outcodeDetails) => {
-      if (outcodeDetails) {
+      if (outcodeDetails && outsideEngland(outcodeDetails)) {
+        log.info('validate-outcode-notEnglish');
+        renderer.postcodeNotEnglish(postcode, req, res);
+        log.info('validate-outcode-notEnglish');
+      } else if (outcodeDetails) {
         res.locals.location = { lat: outcodeDetails.latitude, lon: outcodeDetails.longitude };
         res.locals.isOutcode = true;
         log.info(`outcode ${JSON.stringify(res.locals.location)}`);
         next();
       } else if (!err) {
         log.info('validate-outcode-invalid');
-        postcodeValidator.renderInvalidPostcodePage(postcode, req, res);
+        renderer.invalidPostcodePage(postcode, req, res);
         log.info('validate-outcode-end');
       } else {
         log.info('lookup-outcode-error');
-        postcodeValidator.handlePostcodeError(err, outcodeDetails, res, next);
+        renderer.handlePostcodeError(err, outcodeDetails, res, next);
```

Comment:
This suggests that these branches of code are not covered by current testing. I'll take a look and add some tests then fix.